package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@quero/guidelines",
+  "version": "1.0.0",
+  "description": "The definitive code-style guide by Quero Educação",
+  "engines": {
+    "node": "14",
+    "npm": "7"
+  },
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "lint": "eslint --ext .js --fix --no-eslintrc -c .eslintrc.js ./packages/eslint-config-typescript",
```

Comment:
ah, esse `--no-eslintrc -c .eslintrc.js` parece meio contraditório né <img src="https://cdn.betterttv.net/emote/5ea831f074046462f768097a/3x" width="24px" />

acontece que varios pacotes aqui possuem arquivos `.eslintrc`, e o eslint vai meio que juntando esses caras, para casos de regras de lint especificas para alguma parte do projeto, ou qualuqer coisa do tipo:
https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy

Como não queremos isso, esse  cara fala pra ele ignorar todos esses arquivos e utilizar somente o que estamos enviando por parâmetro.